website/docs/reference/opspec/op.yml/call/container/image.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Image
+---
+
+An image object defines the image to start a container with.
+
+## Properties
+
+### `ref`
+
+_required_
+
+A string referencing a local or remote image.
+
+#### OCI images
```

Comment:
This isn't clear to me if this is a property or what

Reply:
It's a sub-heading of the `ref` property - there are two types, these two headings describe the two types.

Reply:
Yeah I understand but that's not how it displays if you look at the rendered markdown; that's why it used to be a list of references to the actual definitions cuz then you don't have this ambiguity. I kinda think I prefer the former setup linking to the definitions for this reason.

Reply:
I've been reading through these and I think what I'd suggest is a list of properties under the "properties" heading that links to each sub-heading.  I think that would add clarity that "these are the properties, this subset is required, and here is a more in-depth description of each property"